doc/sphinx/addendum/program.rst

```diff
@@ -273,10 +273,17 @@ optional tactic is replaced by the default one if not specified.
    automatically, whether to solve them or when starting to prove one,
    e.g. using :cmd:`Next Obligation`.
 
-   This command supports the :attr:`local` and :attr:`global` attributes.
+   This command supports the :attr:`local`, :attr:`export` and :attr:`global` attributes.
    :attr:`local` makes the setting last only for the current
    module. :attr:`local` is the default inside sections while :attr:`global`
-   otherwise.
+   otherwise. :attr:`export` and :attr:`global` may be used together.
+
+   With no attribute and when :attr:`global` is not combined with
+   :attr:`export`, the meaning is different from the usual meaning of
+   :attr:`global`: the command's effect persists after the current
```

Comment:
"effect persists" singular or "effects persist" plural?

Reply:
could be either without loss of generality